.github/dependabot.yml

```diff
@@ -0,0 +1,38 @@
+# See Dependabot documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "docker"
+    directory: "/.ci"
+    schedule:
+      interval: "weekly"
+    reviewers:
+      - "stuartmorgan"
+      - "ditman"
+    labels:
+      - "ecosystem"
+      - "waiting for tree to go green"
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    reviewers:
+      - "hixie"
+      - "godofredoc"
+    labels:
+      - "team"
+      - "ecosystem"
+      - "team: infra"
+      - "waiting for tree to go green"
+  - package-ecosystem: "gradle" 
```

Comment:
The only gradle dependencies we should have in this repository are in `example/` directories. Is there really value in having those constantly updated by a bot?

Reply:
This is to comply with security good practices of updating dependencies frequently. It may not be extremely relevant for this repo but we would like to run the same security policies across repos of the Dart and Flutter GitHub orgs.

Reply:
I agree.  I also suspect this is hard in Flutter (in general), since it's not very common for Flutter apps to set their own Gradle dependencies.  This might be more valuable in plugin repositories though.

Reply:
Yeah the change in flutter can not be run directly by github as it requires to run additional scripts.

Reply:
> It may not be extremely relevant for this repo but we would like to run the same security policies across repos of the Dart and Flutter GitHub orgs.

We're not going to be able to run it in flutter/plugins without significant changes, because changing the actual plugin dependencies without changelog entries and version bumps will violate policies that we have explicitly established to minimize issues with unexpected breakage.

If we want to auto-manage Gradle dependencies across all of our repos we need a design doc about that where we can discuss the complexities.

Reply:
Removed the gradle update logic and I'll make an entry in the security log to discuss dependency updates.